spring-xd-messagebus-local/src/main/resources/META-INF/spring-xd/bus/local-bus.xml

```diff
@@ -1,15 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <beans xmlns="http://www.springframework.org/schema/beans"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:int="http://www.springframework.org/schema/integration"
-	xsi:schemaLocation="http://www.springframework.org/schema/integration http://www.springframework.org/schema/integration/spring-integration.xsd
+	   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	   xmlns:int="http://www.springframework.org/schema/integration"
+	   xsi:schemaLocation="http://www.springframework.org/schema/integration http://www.springframework.org/schema/integration/spring-integration.xsd
 		http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd">
 
-	<bean id="messageBus" class="org.springframework.xd.dirt.integration.bus.local.LocalMessageBus">
-		<property name="queueSize"
-			value="${xd.local.transport.named.queueSize: #{T(Integer).MAX_VALUE}}" />
-		<property name="poller">
-			<int:poller fixed-rate="${xd.local.transport.named.polling:1000}" />
-		</property>
-	</bean>
-
+	<beans profile="singlenode">
```

Comment:
That's removed in the latest commit

On Wed, Feb 25, 2015 at 12:49 PM, Eric Bottard <notifications@github.com>
wrote:

> In
> spring-xd-messagebus-local/src/main/resources/META-INF/spring-xd/bus/local-bus.xml
> <https://github.com/spring-projects/spring-xd/pull/1471#discussion_r25361387>
> :
>
> >  		http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd">
> >
> > -	<bean id="messageBus" class="org.springframework.xd.dirt.integration.bus.local.LocalMessageBus">
> > -		<property name="queueSize"
> > -			value="${xd.local.transport.named.queueSize: #{T(Integer).MAX_VALUE}}" />
> > -		<property name="poller">
> > -			<int:poller fixed-rate="${xd.local.transport.named.polling:1000}" />
> > -		</property>
> > -	</bean>
> > -
> > +	<beans profile="singlenode">
>
> And then, the next question is: why was the addition of that profile to
> JobPluginTests.java needed
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spring-projects/spring-xd/pull/1471/files#r25361387>.
>